jax/experimental/jax2tf/examples_eval/converters_results.md

```diff
@@ -16,11 +16,11 @@ These exampls are representative for what the average ML researcher is intereste
 | lm1b | SUCCESS |
 | mnist | SUCCESS |
 | nlp_seq | SUCCESS |
-| pixelcnn++ | SUCCESS |
+| pixelcnn++ | FAIL | ValueError('Numerical difference JAX vs TF: JAX result=[[[-1.42588705e-01  4.43906128e-01 -4.43267524e-01 ...  2.41633713e-01\n    4.30841953e-01 -3.73984545e-01]\n  [-5.50750673e-01  8.27922702e-01 -5.37640810e-01 ...  7.13608935e-02\n    5.00771403... (CROPPED)
```

Comment:
I am curious if these discrepancies arise also with enable_xla=True.

Reply:
I added a new converter that just does jax2tf with `enable_xla=True`, PTAL.